pyaedt/hfss3dlayout.py

```diff
@@ -882,7 +886,7 @@ def create_linear_step_sweep(
         >>> oModule.AddSweep
         """
         if sweep_type not in ["Discrete", "Interpolating", "Fast"]:
-            raise AttributeError("Invalid `sweep_type`. It has to be either 'Discrete', 'Interpolating', or 'Fast'")
+            raise AttributeError("Invalid value for `sweep_type`. The value must be `Discrete`, `Interpolating`, or `Fast`.")
```

Comment:
you changed this AttributeError message but accordingly you should change the unit test that checks the message. test_18b_create_linear_step_sweep in test_41_3dlayout_modeler.py in _unittest folder

Reply:
@maxcapodi78 Thank you so much for the lesson. I will to fix the remaining issues up this morning!